hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -169,6 +161,27 @@
         startCleanupThread();
     }
 
+    private OperationExecutor initOperationExecutor() {
+        return new ProgressiveOperationExecutor(
```

Comment:
the progressive executor should be used only when the group property is explicitly enabled, shouldn't it?